projects/fsastorefrontlib/src/cms-components/checkout/components/personal-details-navigation/personal-details-navigation.component.ts

```diff
@@ -24,7 +24,8 @@ export class PersonalDetailsNavigationComponent implements OnInit, OnDestroy {
     protected quoteService: QuoteService,
     protected pricingService: PricingService,
     protected userAccountFacade: UserAccountFacade,
-    protected addressService: FSAddressService
+    protected addressService: FSAddressService,
+    protected userAddressService: UserAddressService
```

Comment:
I think there is no need to inject UserAddressService, because FSAddressService extends it, so you could retrieve addresses also over FSAddressService.